src/tests/test_piranha_java.rs

```diff
@@ -463,7 +463,10 @@ fn test_multiple_code_bases() {
   // Note that we expect 2 matches because we have 2 code bases, and each code base has 1 match.
   // We also have another codebase `folder_3` but the `paths_to_codebase` does not include it.
   assert_eq!(output_summaries.len(), 2);
-  assert_frequency_for_matches(&output_summaries, &HashMap::from([("match_import", 2)]));
+  assert_frequency_for_matches(&output_summaries, &HashMap::from([("match_import", 4)]));
```

Comment:
The snippet

```
package org.piranha.examples;

import java.util.List;
```

```
[0, 0] - [4, 0]
[package_declaration] [0, 0] - [0, 29]
[scoped_identifier] [0, 8] - [0, 28]
    scope: [scoped_identifier] [0, 8] - [0, 19]
        scope: [identifier] [0, 8] - [0, 11]
        name: [identifier] [0, 12] - [0, 19]
   name: [identifier] [0, 20] - [0, 28]
[import_declaration] [2, 0] - [2, 22]
    [scoped_identifier] [2, 7] - [2, 21]
        scope: [scoped_identifier] [2, 7] - [2, 16]
            scope: [identifier] [2, 7] - [2, 11]
            name: [identifier] [2, 12] - [2, 16]
    name: [identifier] [2, 17] - [2, 21]
```

Now has two ways to match. Entire the entire sequential children of the `import_declaration` node, or the last node of the `program` node